appinventor/components/src/com/google/appinventor/components/runtime/Form.java

```diff
@@ -2393,6 +2397,20 @@ private void showAboutApplicationNotification() {
     Notifier.oneButtonAlert(this, message, title, buttonText);
   }
 
+  public void getComponent(View component,ContextMenu menu){
+    componentMap.put(component,menu);
+  }
+
+  @Override
+  public void onCreateContextMenu(android.view.ContextMenu menu, View v, android.view.ContextMenu.ContextMenuInfo menuInfo) {
```

Comment:
You can import `android.view.ContextMenu.ContextMenuInfo` and not have to use it fully qualified here.